recipes/djinni-support-lib/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,11 @@
+#include <djinni_common.hpp>
+#include <cstdlib>
+
+// since we would either need objective C++ or Java (jni) there is not a lot to test
+// just that we have the hearders and can link
```

Comment:
It does not link to anything since nothing is used... Look at your project I do not see anything 💭

Reply:
It includes one file and links on the command line, so at least the lib is found. not enough ?

Reply:
I resolve that topic, we use include files, and link the library. Using the library would mean to either test the Objective-C runtime, or the Java runtime, or both. What is not good to pull in for this purpose.